lib/cli.js

```diff
@@ -182,6 +183,19 @@ const cli = {
 
             log.info(JSON.stringify(fileConfig, null, "  "));
             return 0;
+        } else if (currentOptions.info) {
+            return new Promise(resolve => {
```

Comment:
I think we also need to handle rejections here and not swallow errors from `envinfo`.